larray/tests/test_la.py

```diff
@@ -2855,6 +2860,59 @@ def test_readcsv(self):
         assert_array_equal(la[x.arr[1], 0, 'F', x.nat[1], :],
                            [3722, 3395, 3347])
 
+    def test_read_excel_xlwings(self):
+        if xw is not None:
```

Comment:
We should rather use something like:
```python
    @unittest.skipIf(xw is None, "xlwings is not available")
    def test_read_excel_xlwings(self):
        pass
```
See https://docs.python.org/2/library/unittest.html#skipping-tests-and-expected-failures